README.md

```diff
@@ -49,7 +49,7 @@ If you have questions or stuck, please use the `windows` communication channel o
 <img align="right" src="https://github.com/turanszkij/wickedengine-gifs/raw/main/character_grass.gif" width="320px"/>
 
 #### Linux
-To build the engine for Linux, use Cmake. You can find a sample build script for Ubuntu [here](.github/workflows/build.yml) (in the linux section). On the Linux operating system, you will need to ensure some additional dependencies are installed, such as Cmake (3.7 or newer), g++ compiler (C++ 17 compliant version) and SDL2. For Ubuntu 20.04, you can use the following commands to install dependencies:
+To build the engine for Linux, use Cmake. You can find a sample build script for Ubuntu [here](.github/workflows/build.yml) (in the linux section). On the Linux operating system, you will need to ensure some additional dependencies are installed, such as Cmake (3.7 or newer), g++ compiler (C++ 17 compliant version), zenity and SDL2. For Ubuntu 20.04, you can use the following commands to install dependencies:
```

Comment:
As you said, zenity isn't required. It only seems to be required on KDE, and even then it's not a hard dependency.

I would at least add a note on why zenity is required. It seemingly comes out of the blue. [Add "refuses to elaborate, leaves" meme here]

Reply:
OK tweaked. Even though it's not really _that_ "optional", since _every_ user realistically would like to _not_ silently miss errors or warnings that Wicked wants to tell them. And it's not missing or needed "only on KDE", rather it's just not pre-installed on non-GNOME Linuxes, of which there are _plenty_ out there and all with non-zero user bases. But whatev, I tweaked it, so there :grin:

Reply:
are you sure about the zenity requirement? It seems to me that it is required for dialog boxes, but not message boxes. And the messagebox I see is definitely neither GTK nor Zenity:

![grafik](https://github.com/user-attachments/assets/7a4cb2bb-b782-43dd-b8da-ced3b55102d3)

Reply:
> are you sure about the zenity requirement? It seems to me that it is 

My surety merely stems from what `SDL_GetError` reported when my `SDL_ShowSimpleMessageBox` call returned `-1`. I was surprised myself.. given what they stated in their docs. Sometimes docs are wrong, sometimes error messages are wrong..

> Edit: oh, the "implementation" for wayland is calling zenity. kek. That's pretty stupid, I agree

Even odder since I'm on X11 :grin: